package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Dec 12 10:44:17 UTC 2018 - mfilka@suse.com
+
+- bnc#1116568
+  - /etc/hosts load performance improvements
```

Comment:
NP: Usually our team mates encourage me to write less technical log entries. So, I guess something like `Improve the performance loading the /etc/hosts, avoiding to crash when dealing with quite long files (bsc#...)` could be better. But, it is only a suggestion. Remember that, sadly, English is not one of my strengths :wink: